src/diagnostics.rs

```diff
@@ -132,8 +132,6 @@ pub fn report_error<'tcx, 'mir>(
             };
             #[rustfmt::skip]
             let helps = match e.kind() {
-                Unsupported(UnsupportedOpInfo::NoMirFor(..)) =>
```

Comment:
This error is never raised by Miri anyway, and I will try to remove this error variant from rustc.